src/poprox_concepts/domain/profile.py

```diff
@@ -1,14 +1,13 @@
-from typing import Dict, List, Optional
 from uuid import UUID
 
 from pydantic import BaseModel
 
-
-from poprox_concepts.domain.click_history import ClickHistory
 from poprox_concepts.domain.account import AccountInterest
+from poprox_concepts.domain.click_history import ClickHistory
 
 
 class InterestProfile(BaseModel):
+    profile_id: UUID | None = None
     click_history: ClickHistory
-    click_topic_counts: Optional[Dict[str, int]] = None
-    onboarding_topics: List[AccountInterest]
+    click_topic_counts: dict[str, int] | None = None
```

Comment:
Is there an actual pro-con on `Optional` vs `| None` for this use case?
Not blocking, just hoping to learn.

Reply:
Mostly just that it passes the linter/formatter that @mdekstrand set up for `poprox-recommender`. Apparently `Optional` is deprecated and `| None` is the new hotness. (I had no idea until `ruff` complained.)